### DISCUSSION TITLE
Mac Problem

### DISCUSSION THREAD

**Discussion by laerttt** [NvChad/NvChad#2403] [General]
I am new to this, can some body explain why does it look like this on the mac terminal, and how do i fix it?
![Screenshot 2023-09-25 at 11 46 09 PM](https://github.com/NvChad/NvChad/assets/101653303/1faa9c90-719e-4e61-99bb-02e25c574a69)

**Comment by siduck** [comment]
please use true colors in your terminal or use a terminal that does it by default like kitty alacritty wezterm

**Reply by laerttt** [reply]
Thanks
